korangar/src/main.rs

```diff
@@ -2417,5 +2437,6 @@ impl ApplicationHandler for Client {
 
     fn suspended(&mut self, _event_loop: &ActiveEventLoop) {
         self.graphics_engine.on_suspended();
+        self.audio_engine.on_suspended();
```

Comment:
The graphic engine only has a "on_suspend()" because it needs to be tightly integrated into the windows / graphic output loop. No need to couple the audio engine too.

Reply:
The on_suspend depend on the operating system, in some operating system may continue to reproduce the sound during on_suspend. Maybe on Wayland, X11, I am not sure. If you want, I can remove it.

Reply:
That's not what I tried to say. I just said that the audio engine itself doesn't need to know the concept of "a windows suspended". That is something the application can handle and call the audio engine functionality directly.

Reply:
Oh, I misunderstand